text/0031-integ-testing.md

```diff
@@ -0,0 +1,300 @@
+# Integration testing for CDK apps
+
+* **Original Author(s):**: [@nija-at](https://github.com/nija-at)
+* **Tracking Issue**: [#31](https://github.com/aws/aws-cdk-rfcs/issues/31)
+* **API Bar Raiser**: @{BAR_RAISER_USER}
+
+CDK app developers and construct library authors can write integration
+tests that can be run locally or in their CI/CD pipeline.
+
+## Working Backwards - README.md
+
+### contest
+
+🌩️ contest tests can verify that your CDK app can be deployed successfully on AWS with the desired resources.
```

Comment:
Are we also taking care of cleanup? I think that'd be a selling point.

Reply:
Agreed. I'll add.